test/features/calendar.feature

```diff
@@ -115,6 +117,8 @@ Feature: calendar module
             | endDate     | "2021-01-01T00:00:00.000Z"            |
         And the response should contain:
             | documentId | "TYPE:ID" |
+        Then set header "User-Uid" with value "hubber"
+        Then set header "Platform" with value "discord"
```

Comment:
We probably need another test to check when the headers do not match that there is a failure and no update occurs